content/en/docs/ops/best-practices/observability/index.md

```diff
@@ -0,0 +1,271 @@
+[Istio Standard Metrics](/docs/reference/config/telemetry/metrics/) and renaming any metrics collected by removing
+the prefix used in the workload-level recording rules (`workload:`). This will allow existing dashboards and
+queries to seamlessly continue working when pointed at the production Prometheus instance (and away from the Istio instance).
+
+Control plane metrics are also collected and federated up to the production Prometheus.
+{{< /tip >}}
+
+### Optimizing metrics collection with recording rules
+
+Beyond just using recording rules to [aggregate over pods and instances](#workload-level-aggregation-via-recording-rules), you may
+want to use recording rules to generate aggregated metrics tailored specifically to your existing dashboards and alerts. Optimizing
+your collection in this manner can result in large savings in resource consumption in your production instance of Prometheus, in
+addition to faster query performance.
+
+For example, imagine a custom monitoring dashboard that used the following Prometheus queries:
+
+* Total rate of requests averaged over the past minute by destination service name and namespace
+
+    {{< text plain >}}
+    sum(irate(istio_requests_total{reporter="source"}[1m]))
+    by (
+        destination_canonical_service,
+        destination_workload_namespace
+    )
+    {{< /text >}}
+
+* P95 client latency averaged over the past minute by source and destination service names and namespace
+
+    {{< text plain >}}
+    histogram_quantile(0.95,
+      sum(irate(istio_request_duration_milliseconds_bucket{reporter="source"}[1m]))
+      by (
+        destination_canonical_service,
+        destination_workload_namespace,
+        source_canonical_service,
+        source_workload_namespace,
+        le
+      )
+    )
+    {{< /text >}}
+
+The following set of recording rules could be added to the Istio Prometheus configuration, using the `istio` prefix
+to make identifying these metrics for federation simple.
+
+{{< text yaml >}}
+groups:
+- name: "istio.recording-rules"
+  interval: 5s
+  rules:
+  - record: "istio:istio_requests:by_destination_service:rate1m"
+    expr: |
+      sum(irate(istio_requests_total{reporter="destination"}[1m]))
+      by (
+        destination_canonical_service,
+        destination_workload_namespace
+      )
+  - record: "istio:istio_request_duration_milliseconds_bucket:p95:rate1m"
+    expr: |
+      histogram_quantile(0.95,
+        sum(irate(istio_request_duration_milliseconds_bucket{reporter="source"}[1m]))
+        by (
+          destination_canonical_service,
+          destination_workload_namespace,
+          source_canonical_service,
+          source_workload_namespace,
+          le
+        )
+      )
+{{< /text>}}
+
+The production instance of Prometheus would then be updated to federate from the Istio instance with:
+
+* match clause of `{__name__=~"istio:(.*)"}`
```

Comment:
Hmm the spell checker has more issues with ` characters than I thought. How about changing lines 258-266 to the following, as a temporary workaround:
```
{{< text plain >}}
- match clause of {__name__=~"istio:(.*)"}
- metric relabeling config with: regex: "istio:(.*)"
{{< /text >}}

The original queries would then be replaced with:

{{< text plain >}}
- istio_requests:by_destination_service:rate1m
- avg(istio_request_duration_milliseconds_bucket:p95:rate1m)
{{< /text >}}
```

Reply:
Never mind, I found the actual problem. See comment on line 254.